addons/l10n_in_tcs_tds/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_in_pan = fields.Char(string="PAN", help="A permanent account number is a ten-character alphanumeric identifier, by the Indian Income Tax Department")
```

Comment:
When is this number needed?

Reply:
the company need to report government that the company cut TAX DEDUCTED AT SOURCE (TDS) against this PAN

Reply:
The idea would be to describe that in the help msg
```suggestion
    l10n_in_pan = fields.Char(string="PAN", help="A permanent account number is a ten-character alphanumeric identifier, by the Indian Income Tax Department for e.g. the TDS")
```

Reply:
```suggestion
    l10n_in_pan = fields.Char(string="PAN", help="The permanent account number is a ten-character alphanumeric identifier, needed by the Indian Income Tax Department for the TDS report (for example)")
```
? :thinking:

Reply:
@jco-odoo, I have added a new help message below:
PAN enables the department to link all transactions of the “person” with the department. 
These transactions include tax payments, TDS/TCS credits, returns of income/wealth/gift/FBT, specified transactions, correspondence, and so on.
thus, PAN acts as an identifier for the “person” with the tax department. 


I hope it will make sense to the user now.

Reply:
@jco-odoo, for now, can you check the help message on the PAN field?
I have edited it as per the Indian income tax department's definition of PAN.
let me know if there are any changes required!